File: MiniDOMWebLibs1/MiniDOM/Collegamenti/UserAllowNegate/LinkUserAllowNegate.vb
Imports minidom
Imports minidom.Sistema
Imports minidom.Databases

Partial Public Class WebSite

    <Serializable> _
    Public Class LinkUserAllowNegate
        Inherits UserAllowNegate(Of CCollegamento)

        Public Sub New()
        End Sub

        Protected Overrides Function GetItemFieldName() As String
            Return "Collegamento"
        End Function

        Protected Overrides Function GetUserFieldName() As String
            Return "Utente"
        End Function

        Public Overrides Function GetTableName() As String
            Return "tbl_CollegamentiXUtente"
        End Function

        Protected Overrides Function GetConnection() As CDBConnection
            Return Databases.APPConn
        End Function

        Public Overrides Function GetModule() As CModule
            Return Nothing
        End Function

    End Class

End Class

File: MiniDOMWebLibs1/MiniDOM/Collegamenti/UserAllowNegate/LinkUserAllowNegate.cs
using System;

namespace minidom
{
    public partial class WebSite
    {
        [Serializable]
        public class LinkUserAllowNegate : UserAllowNegate<CCollegamento>
        {
            public LinkUserAllowNegate()
            {
            }

            protected override string GetItemFieldName()
            {
                return "Collegamento";
            }

            protected override string GetUserFieldName()
            {
                return "Utente";
            }

            public override string GetTableName()
            {
                return "tbl_CollegamentiXUtente";
            }

            protected override Databases.CDBConnection GetConnection()
            {
                return Databases.APPConn;
            }

            public override Sistema.CModule GetModule()
            {
                return null;
            }
        }
    }
}